src/Sylius/Bundle/ShopBundle/templates/account/reset_password.html.twig

```diff
@@ -1,9 +1,12 @@
 {% extends '@SyliusShop/shared/layout/base.html.twig' %}
 
+{% set reset_password_hook = 'sylius_shop.account.reset_password' %}
+{% set prefixes = [reset_password_hook] %}
```

Comment:
Is it used?

Reply:
It is in `@SyliusShop/shared/layout/base.html.twig` template